HealthcareManagement.IntegrationTests/AppointmentsControllerIntegrationTests.cs

```diff
@@ -39,6 +44,15 @@ public AppointmentsControllerIntegrationTests(WebApplicationFactory<Program> fac
                         options.UseInMemoryDatabase("InMemoryDbForTesting");
                     });
                 });
+                
```

Comment:
Ar trebui vazut daca iti merg testele de integrare fara a modifica aici, cel putin la mine merge daca nu adaug si modificarile din fisierul asta ( adica sa las doar modificarea cu ?? ) . Problema este ca in testele de integrare nu se incarca env-ul

Reply:
Îmi merg și mie fără modificările din fișierul ăsta, dar la fel, când se păstrează ?? În program.cs